apps/judicial-system/backend/infra/judicial-system-backend.ts

```diff
@@ -81,3 +81,7 @@ export const serviceSetup = (): ServiceBuilder<'judicial-system-backend'> =>
     .liveness('/liveness')
     .readiness('/liveness')
     .postgres(postgresInfo)
+    .resources({
+      requests: { cpu: '750m', memory: '750Mi' },
+      limits: { cpu: '1000m', memory: '1Gi' },
```

Comment:
Should be:
requests: { cpu: '100m', memory: '256Mi' },
 limits: { cpu: '400m', memory: '512Mii' },